modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -81,11 +83,18 @@ def create_draft
 
         slots = fetch_provider_slots(cached_referral_data, provider.id)
         draft = eps_appointment_service.create_draft_appointment(referral_id:)
+
+        unless draft&.id
```

Comment:
I feel you need unit testing for at least this. All the other Statsd calls should be tested too.